android/libraries/rib-base/src/main/kotlin/com/uber/rib/core/RibComponentType.kt

```diff
@@ -13,8 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.uber.rib.workers.root.main.workers.monitoring
+package com.uber.rib.core
 
-fun interface BackendReporter {
-  fun report(genericMessage: String)
+public enum class RibComponentType {
```

Comment:
Regular Worker?

Reply:
We may need to tweak a bit more RibCoroutine worker in order to provide accurate monitoring as mentioned by @psteiger earlier 

Given there are barely any users internally of RibCoroutineWorker, I'm not concerned to leave these changes to be implemented on a separate PR